packages/apollos-ui-mapview/src/MapView/index.js

```diff
@@ -41,6 +41,11 @@ const StyledCampusCard = styled(
   'ui-mapview.MapView.StyledCampusCard'
 )(CampusCard);
 
+const StyledConfirmButton = withTheme(
```

Comment:
Can we just call this the `ConfirmButton` since it's not actually using `styled`?
```suggestion
const ConfirmButton = withTheme(
```